errbot/errBot.py

```diff
@@ -265,6 +266,18 @@ def process_message(self, mess):
     def _process_command(self, mess, cmd, args, match):
         """Process and execute a bot command"""
 
+        # first it must go through the command filters
+        try:
+            for cmd_filter in self.command_filters:
+                mess, cmd, args = cmd_filter(mess, cmd, args)
+                if mess is None:
+                    log.info("Command %s deferred." % cmd)
+                    return
+        except ACLViolation as e:
```

Comment:
Ha you are right. I gonna push the ACLException in the acls plugin with directly a response to the messsage and just let the if none logic here (that block the message basically).